packages/storybook/stories/Examples/TeamsInterop/TeamsInteropDocs.tsx

```diff
@@ -32,12 +32,13 @@ export const getDocs: () => JSX.Element = () => {
       <Source code={LobbyComponentText} />
       <Heading>Inline Images</Heading>
       <SingleLineBetaBanner />
+      <Description>Microsoft Teams allows users to share images by:</Description>
+      <Description>- Copying & paste into the box at the bottom of the chat.</Description>
```

Comment:
Change "box at the bottom" to "input box"?

Reply:
it's a very specific wording copied from Teams website